### PR TITLE
make status_justify configurable

### PR DESCRIPTION
**Pull request by anopheles**

### CODE REVIEW

**Comment by vdbe** [comment]
The default for `status-justify` is left, would it not be better to just remove the set line for it and let users set it?
You can also just overwrite it.

**Comment by anopheles** [comment]
I would only do that if I were confident that this theme works well with every justify setting. Otherwise an existing tmux config may look broken. Making this setting explicit moves the responsibility to the user. Otherwise, totally agree! 👍

**Review by sgoudham — CHANGES_REQUESTED** [review]
Hi :wave: 

Thanks for the PR, I think it has merit. 

I believe @89iuv would also want you to update the documentation to explicitly mention this option in the [README#Status](https://github.com/catppuccin/tmux?tab=readme-ov-file#status)

File: catppuccin.tmux [hunk 1]
```diff
@@ -325,7 +325,8 @@ main() {
   local status_default=$(get_tmux_option "@catppuccin_status_default" "on")
   set status "$status_default"
   set status-bg "${thm_bg}"
-  set status-justify "left"
+  local status_justify=$(get_tmux_option "@catppuccin_status_justify" "left")
```
[the remark this answers is not in the document]
Comment (anopheles) [reply catppuccin.tmux:326]: done

[the remark this answers is not in the document]
Comment (vdbe) [reply catppuccin.tmux:326]:
```bash
  # status
  local status_default=$(get_tmux_option "@catppuccin_status_default" "on")
  local status_justify=$(get_tmux_option "@catppuccin_status_justify" "left")

  set status "$status_default"
  set status-justify "$status_justify"
  set status-bg "${thm_bg}"
  set status-left-length "100"
  set status-right-length "100"
```

**Review by sgoudham — APPROVED** [review]

**Review by vdbe — APPROVED** [review]

**Comment by Danisaski** [comment]
Why was this feature reverted?

**Comment by kjnsn** [comment]
> Why was this feature reverted?

It's a tmux option. There's no value in configuring through catppuccin, just set the option to whatever you'd like

**Comment by Danisaski** [comment]
> > Why was this feature reverted?
> 
> It's a tmux option. There's no value in configuring through catppuccin, just set the option to whatever you'd like

Thanks for the quick reply!

**Pull request merged**